coreff_pappers/pappers.py

```diff
@@ -0,0 +1,237 @@
+import requests
+import json
+import base64
+from odoo.exceptions import UserError
+
+
+def clean_code(code):
+    """Removes any non-numerical character and returns a string"""
+    cleaned_code = ""
+    for character in code:
+        if character.isdigit():
+            cleaned_code += character
+    return cleaned_code
```

Comment:
need to be check_code instead of clean_code.
This function raise error in code not good